src/components/modebar/manage.js

```diff
@@ -8,6 +8,7 @@ var isUnifiedHover = require('../fx/helpers').isUnifiedHover;
 var createModeBar = require('./modebar');
 var modeBarButtons = require('./buttons');
 var DRAW_MODES = require('./constants').DRAW_MODES;
+var cloneDeep = require('lodash').cloneDeep;
```

Comment:
Thanks very much for the PR.
Please note that [`lodash`](https://www.npmjs.com/package/lodash) is a dev-dependency. 
And for various reasons (e.g. size & potential security issues) we are not interested to include that in our bundles.
Alternatively you could use
```js
var extendDeep = require('../../lib').extendDeep;
```
and then 
```js
var customButtons = extendDeep({}, context.modeBarButtons);
```
Or perhaps even better would be to use `extendFlat` if possible?

Reply:
Fixed. Using the extendFlat function is not possible because the object has a nested structure.